lib/driver.js

```diff
@@ -251,6 +255,31 @@ class AndroidDriver extends BaseDriver {
     await this.initAutoWebview();
   }
 
+  shouldDismissChromeWelcome (opts) {
```

Comment:
yeah, either we should use the instance var, or this method should just be a non-member function outside of the class definition.